src/main/java/hudson/plugins/plot/Plot.java

```diff
@@ -538,32 +543,74 @@ public void plotGraphMap(StaplerRequest req, StaplerResponse rsp)
      * Called when a build completes. Adds the finished build to this plot. This
      * method extracts the data for each data series from the build and saves it
      * in the plot's CSV file.
-     *
-     * @param build
-     * @param logger
      */
     public void addBuild(AbstractBuild<?, ?> build, PrintStream logger) {
```

Comment:
🐜  Wouldn't it be better to remove duplicate logic and just invoke the version taking a Run?  It seems like a lot of this could be condensed.

Reply:
(ant designates a nitpick, i.e. ant == nit)